docs/howtoguides/check_timer_jobs_config.md

```diff
@@ -0,0 +1,15 @@
+# How to check timer jobs configuration
```

Comment:
Could we merge this and `changing_a_timer_job_interval` into something like `configuring_timer_jobs`?

Reply:
Good point